enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/plugin/PluginRealm.java

```diff
@@ -148,8 +149,16 @@ protected AuthorizationInfo doGetAuthorizationInfo( PrincipalCollection principa
     {
         if ( authorizationPlugin != null )
         {
-            org.neo4j.server.security.enterprise.auth.plugin.spi.AuthorizationInfo authorizationInfo =
-                    authorizationPlugin.authorize( getPrincipalAndRealmCollection( principals ) );
+            org.neo4j.server.security.enterprise.auth.plugin.spi.AuthorizationInfo authorizationInfo;
```

Comment:
Why does it have to contain the full qualified name here? Can we rename it to avoid this?